### DISCUSSION TITLE
RfC: binding agent and cell colonies

### DISCUSSION THREAD

**Discussion by schnautzr** [Revolutionary-Games/Thrive#3610] [Q&A] [answered]
I recently proposed a change to the locale that was met with uncertainty, so let's discuss.

I did some research, and as best I can tell, Thrive's current **binding agent** appears to work the same way that amoebas are known to work. Here's a crash-course for you.

At the most basic level: Certain unicellular organisms are known to use a binding agent to themselves sticky and join together to form a larger pseudo-organism, which is capable of either breaking apart again or actually altering the functions members of it carry out.

Since Thrive uses a lot of biological terms, it seems appropriate we do so. Let's explore what's actually happening here:

The organisms in question are amoebas belonging to a group known as slime molds. The amoebas release a pheromone called **acrasin**, which is an adhesive signaling compound which causes other amoebas to adhere to them. The colonial structure they form is known as a **grex** or **slug** (depending on the literature). The grex functions as a unit and may either break apart again or **differentiate** (meaning the individual members become altered such that they carry new functions). It is thought that this is one way that multicellular organisms evolved. As far as I can tell, scientists haven't identified yet how or where the acrasin is produced.

This being the only LAWK version I can identify of a "binding agent", I'm proposing we add relevant scientific information in the locale files to help users understand what is actually happening here, because as it is, it appears we're just having the cell make glue or something, which feels really silly without any informational background.

**Comment by hhyyrylainen** [comment, marked as answer]
This should be posted as an issue, otherwise only I would notice this even exists.

Really briefly I'm gainst the terms acrasin and grex as they seem to be slime mould exclusive terms, which we don't have in the game.

**Reply by schnautzr** [reply]
I'll repost there then.

**Reply by schnautzr** [reply]
#3612

**Reply by athariqk** [reply]
> This should be posted as an issue, otherwise only I would notice this even exists.

Just FYI, it looks like this discussion post appeared in the dev Discord's (and community Discord's) github feed for me along with the replies, which made me aware this exists.

**Reply by hhyyrylainen** [reply]
I did realize that that's a thing, however at one point our github webhooks weren't even configured to react to the discussion events (they are a totally different category than anything else). Also I think normal watching of a repo doesn't notify you of all the discussions, though I might be wrong on this too.
